diff --git a/configure.ac b/configure.ac
index 962dda0..e7515e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT([Tempo2],[2021.06.1],[george.hobbs@csiro.au],[],[http://www.bitbucket.org/psrsoft/tempo2])
+AC_INIT([Tempo2],[2021.07.1],[george.hobbs@csiro.au],[],[http://www.bitbucket.org/psrsoft/tempo2])
 
 AC_ARG_VAR([TEMPO2], [Default TEMPO2 installation directory])
 AC_PREFIX_DEFAULT([${TEMPO2:-"/usr/local"}])
