diff --git a/tempo2.h b/tempo2.h
index 5eb9704..927c61b 100644
--- a/tempo2.h
+++ b/tempo2.h
@@ -50,8 +50,8 @@
 
 
 #define TEMPO2_h_HASH "$Id$"
-#define TEMPO2_h_VER "2021.06.1"
-#define TEMPO2_h_MAJOR_VER 2021.06
+#define TEMPO2_h_VER "2021.07.1"
+#define TEMPO2_h_MAJOR_VER 2021.07
 #define TEMPO2_h_MINOR_VER 1
 #define TSUN longdouble(4.925490947e-6) /*!< Solar constant for mass calculations. */
 #define MAX_FREQ_DERIVATIVES 13    /*!< F0 -> Fn   where n=10                            */
